cmd/api/acrsdk.go

```diff
@@ -0,0 +1,205 @@
+package api
```

Comment:
can't remember if I put them in this repo, but I think we should add copyright headers to all of these files:

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```